webapp/channels/src/components/admin_console/server_logs/logs.tsx

```diff
@@ -144,16 +151,28 @@ export default class Logs extends React.PureComponent<Props, State> {
                         <FormattedMessage {...messages.bannerDesc}/>
                     </div>
                 </div>
-                <button
-                    type='submit'
-                    className='btn btn-primary'
-                    onClick={this.reloadPlain}
-                >
-                    <FormattedMessage
-                        id='admin.logs.ReloadLogs'
-                        defaultMessage='Reload Logs'
-                    />
-                </button>
+                <div className='banner-buttons'>
+                    <button
+                        type='submit'
+                        className='btn btn-primary'
+                        onClick={this.reloadPlain}
+                    >
+                        <FormattedMessage
+                            id='admin.logs.ReloadLogs'
+                            defaultMessage='Reload Logs'
+                        />
+                    </button>
+                    <button
```

Comment:
The reason that this isn't loading the logs now is that, instead of a button that runs JS to fetch the file, you want this to be an anchor element with the href being the API you added.
```
<a
    className='btn btn-primary'
    href={Client4.getUrl() + 'api/v4/logs/download'}
    target='_blank'
>
</a>
```
When you click on that and the browser tries to navigate to it, it'll instead open a new tab and download the log file. At least on Firefox, that worked for me with the server code you had